README.md

```diff
@@ -3,3 +3,45 @@
 Dataset for the paper L. Bonati, R. Shirkhani, C. Fiandrino, S. Maxenti, S. D'Oro, M. Polese, T. Melodia, "Twinning Commercial Network Traces on Experimental Open RAN Platforms," in Proceedings of ACM WiNTECH, Washington, DC, USA, November 2024.
 
 The dataset is available at the following [link](https://repository.library.northeastern.edu/collections/neu:h989sz017). If you use it in your work, please reference the paper above.
+
+# Experimental setup
+- Number of Base Stations (BSs): 1
+- Channel bandwidth: 10 MHz (50 Physical Resource Blocks (PRBs))
+- Number of slices for each BS: 2
+- Scheduling policies available to each slice:
+    - Policy 0: Round-robin (RR)
+    - Policy 2: Proportionally fair (PF)
+- Number of User Equipments (UEs): 8
+    - Users 1-4 demand Enhanced Mobile Broadband (eMBB) traffic
+    - Users 5-8 demand Ultra Reliable and Low Latency Communications (URLLC) traffic
+- UE mobility: static
+- Traffic: The traffic is twinned from a public dataset of LTE network from 3 BSs located in Madrid, Spain.
+    - BS1: carrier frequency at 816 MHz
+    - BS2: carrier frequency at 1835 MHz
+    - BS3: carrier frequency at 2650 MHz
+- Number of slices for the BS: 2 (slice_id 0 for eMBB UEs and slice_id 1 for URLLC UEs)
+- Slicing configurations: There are 50 PRBs in total to allocate for the two slices.
+
+|  Slice Resouce Allocation  | Slice ID 0 (eMBB: UEs 1-4) PRBs  |  Slice ID 1 (URLLC: UEs 5-8) PRBs  |
+|------------|------------|------------|
+| Slicing 1 | 9 | 41 |
+| Slicing 2 | 21 | 29 |
+| Slicing 3 | 30 | 20 |
+| Slicing 4 | 39 | 11 |
+| Slicing 5 | 50 | 0 |
+
+# Dataset stucture
+- The dataset has the following structure: 3 clusters, 5 slicings, 2 schedualing. In total 30 combinations.
```

Comment:
The first bullet point is not easy to understand. Can we elaborate more on this?